src/com/pentalog/pentastagiu/homework2/ConsoleApp/Basket.java

```diff
@@ -0,0 +1,21 @@
+package com.pentalog.pentastagiu.homework2.ConsoleApp;
+
+public class Basket {
+    private static float TotalPrice ;
+    Customer customer;
+
+    public void AddProduct(Product product)
+    {
+        TotalPrice = TotalPrice + product.getPrice();
+
+    }
+    public float getTotalPrice() {
+        if(customer.getMembership()=="GOLD")
+        return (TotalPrice- TotalPrice*(2.0f/10));
+        else
+            if(customer.getMembership()=="SILVER")
```

Comment:
ce se intampla daca cel care creeaza Basket uita sa seteze customerul? o exceptie va fi aruncata aici, de-asta sugeram sa ai field-ul customer declarat private si in constructor sa verifici sa nu fie null